examples/logger.py

```diff
@@ -133,6 +136,29 @@ def __init__(self, name: str, parent: Optional[QObject] = None):
         logger = logging.getLogger()
         logger.setLevel(logging.DEBUG)
         logger.addHandler(self.handler)
+        self.initLevelSelector(self.loggerFrame.levelSelector)
+
+    def initLevelSelector(self, box: QComboBox):
+        """Initializes a combo box for selecting level."""
+        box.addItem("DEBUG")
+        box.addItem("INFO")
+        box.addItem("WARNING")
+        box.addItem("ERROR")
+        box.addItem("CRITICAL")
+        box.activated[str].connect(self.fixLevel)
```

Comment:
Referred to [activated](https://doc.qt.io/qtforpython-5/PySide2/QtWidgets/QComboBox.html#PySide2.QtWidgets.PySide2.QtWidgets.QComboBox.activated), this signal is deprecated.

Please use another function!
I suggest a solution.

```python
box.activated.connect(self.fixLevel)

def fixLevel(self):
    text = self.loggerFrame.levelSelector.currentText()
    ...
```

Reply:
Also please move this line into the constructor!
We usually connect a signal to a slot in there.

Reply:
I am not sure whether signal 'activated' has problem, so I changed to 'currentTextChanged'. It emits signal whenever currentText is changed.
I moved codes to construtor.